docs/src/gettingstarted/introduction/template/cli-requirements.md

```diff
@@ -11,36 +11,16 @@ With the [management console]({{< relref "/administration/web/_index.md" >}}) yo
 
 Before you install it there are a few requirements that must be met first.
 
-### Git
+## Git
 
 [Git]({{< relref "/development/tools.md#git" >}}) is the open source version control system used by Platform.sh. Any change you make to your Platform.sh project will need to be committed via Git. You can see all the Git commit messages of an environment in the `Environment Activity` feed of the [management console]({{< relref "/administration/web/_index.md" >}}) for each project you create.
 
 Before getting started, make sure you [have Git installed](https://git-scm.com/) on your computer.
 
-### SSH key pair
+## SSH
 
-Once your account has been set up and the [CLI]({{< relref "/development/cli/_index.md" >}}) is installed, Platform.sh needs one additional piece of information about your computer so that you can access your projects from the command line.
-If you are unfamiliar with how to generate an SSH public and private key, there are instructions in the documentation about [how to do so]({{< relref "/development/tools.md#ssh" >}}).
+You will need Secure Shell (SSH) to securely connect to your Git repository and environments.  SSH clients are readily available for every platform, and may already be installed on your computer.
```

Comment:
I think we need to bring back here the CLI requirement that we mentioned before. 

Formerly we recommended CLI+Git+SSH Keys, now we recommend CLI + Git. 

Maybe it can be in the paragraph below as: 

> Once your account has been set up and the [CLI]({{< relref "/development/cli/_index.md" >}}) is installed, you can run `platform ssh` to securely connnect to your environments. `platform ssh` relies on certificate-based authentication. 
>
> If you wish to use keypair authentication instead, see the [SSH page]({{< relref "/development/ssh.md" >}})

I think a paragraph like this one this may be simpler to understand than my suggestions in the paragraph below.

Reply:
This page is specifically the prerequisites for the CLI.  The next page in the sequence shows how to install the CLI itself.  And having the SSH client available for the CLI to wrap is a requirement.

Reply:
Oh, missed that context, thank you for clarifying. Somehow I thought the CLI came in first, given the paragraph we had written in the previous version 😅 all good then, thanks again!